presto-product-tests/bin/product-tests-suite-8-non-generic.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I would prefer to name this file `product-tests-suite-hdp3.sh`, but this against the convention.

WDYT @findepi @electrum ?

Reply:
please don't change the convention